salt/config/schemas/esxvm.py

```diff
@@ -139,8 +144,10 @@ class ESXVirtualMachineSerialSchemaItem(ComplexSchemaItem):
     )
     adapter = StringItem(
         title="Virtual Machine Serial Port Name",
-        description="Unique adapter name for virtual machine serial port"
-        "for creation an arbitrary value should be specified",
+        description=(
+            "Unique adapter name for virtual machine serial port"
+            "for creation an arbitrary value should be specified"
```

Comment:
Do we need a space at the start of the second line?

Reply:
Fixed